improver_tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -307,6 +335,191 @@ def test_exception_for_single_entry_cubelist(self):
         with self.assertRaisesRegex(ValueError, msg):
             plugin.process(cubelist, "new_cube_name")
 
+    def test_broadcast_coord(self):
```

Comment:
```suggestion
    def test_broadcast_threshold_coord(self):
```
I think this makes the difference between this test and the `test_broadcast_non_threshold_coord` clearer.

Reply:
Done